hosts/dinar-ephemeral-alpha/default.nix

```diff
@@ -16,22 +16,25 @@ in
     ];
   };
 
-  # Localization
-  networking.hostName = "dinar-ephemeral-alpha";
-  time.timeZone = "Europe/Helsinki";
-
-  boot.kernelPackages = pkgs.linuxPackagesFor (pkgs.linux);
+  # Localization options
+  localization = {
+    hostname = "dinar-ephemeral-alpha";
+    timezone = "Europe/Helsinki";
+    keymap = "fi";
```

Comment:
The keymap option should be a home-manager option on per user basis if declared. I think both Tommi and I use ANSI us keymap.